session_factory.go

```diff
@@ -119,6 +128,10 @@ func (f sessionFactory) newSession(
 		}
 
 		if s.appDataDictionary, err = datadictionary.Parse(dataDictionaryPath); err != nil {
+			err = errors.Wrapf(
```

Comment:
When an error happens here, we don't know which config is wrong neither with file does not exist.